### DISCUSSION TITLE
next GET / 200 disable

### DISCUSSION THREAD

**Discussion by DemetryCode** [vercel/next.js#64726] [Help] [answered]
### Summary

Couldn't find any proper solution. I am not sure since when it started, perhaps a week or two ago, but nextjs started showing in logs "GET / 200" after each render, it looks like that:
```
 ✓ Starting...
 ✓ Ready in 1125ms
 ○ Compiling / ...
 ✓ Compiled / in 1156ms (523 modules)
 GET / 200 in 1434ms
 ✓ Compiled in 191ms (250 modules)
```
Is there a way to disable it?

**Comment by icyJoseph** [comment]
Could you share your  next.js config? there's a setting that enables/disables this

**Reply by DemetryCode** [reply]
It happens all the time even when I create fresh new project with literally  0 changes - it throws me that GET /200.
```
npx create-next-app@latest .
npm run dev
```
so next.config is just like that
```
/** @type {import('next').NextConfig} */
const nextConfig = {};

export default nextConfig;
```

**Reply by icyJoseph** [reply]
Aha, I mixed that one up with cache logging, https://nextjs.org/blog/next-14-1#data-cache-logging

But that, is not throwing, it is just informing you that a GET request to `/` has arrived and served with a 200. Why do you want to silence it?

**Reply by DemetryCode** [reply]
Because I didn't make that request. Why it shows me what I don't ask for. It's kinda weird

**Reply by icyJoseph** [reply]
It is pretty common for servers, at the very least during development mode, to show incoming requests. That's what it is showing. That a client made a request to your server for /, and it was served a 200 response. Once your app grows more you'll be glad to have these.

**Reply by mopatches** [reply]
Hello! Is there a way to disable this new request logging? It looks like it may have been added in https://github.com/vercel/next.js/pull/62946/files but I don't see a way to turn it off.

```
POST /api/... 200 in 13ms
POST /api/... 200 in 15ms
POST /api/... 200 in 21ms
POST /api/... 200 in 15ms
POST /api/... 200 in 23ms
```

Our nextjs server is in frequent communication with various backend services, so the dev logs are now full of check-in spam as the backend services talk with nextjs.

**Reply by mopatches** [reply]
Opened a feature request for that here: https://github.com/vercel/next.js/discussions/65992

**Reply by wesnishio** [reply]
Any updates? I am also flooded with too many API logs and can't find a real log.

**Reply by DemetryCode** [reply, marked as answer]
Yes, you can disable it in next.config now: https://nextjs.org/docs/app/api-reference/next-config-js/logging

**Reply by icyJoseph** [reply]
It was, as I linked months ago hehe

**Reply by DemetryCode** [reply]
no, it was added in Next v15

**Reply by icyJoseph** [reply]
Nah. My link literally pointed to 14.1, https://nextjs.org/blog/next-14-1#data-cache-logging
<img width="1196" alt="Screenshot 2024-12-02 at 16 12 36" src="https://github.com/user-attachments/assets/81a1a96a-6488-4d1b-8497-852942bc61b7">

The feature itself was stabilised (had been experimental), over a year ago, https://github.com/vercel/next.js/commit/4e1429182f50477e396ff4f3a277949521adf866

**Reply by DemetryCode** [reply]
You still have no clue what this all thread was about linking to something not related, if you need more info you can read here https://github.com/vercel/next.js/discussions/65992
